v2.2/cost-based-optimizer.md

```diff
@@ -127,20 +127,18 @@ To turn on this feature:
 
 ## Query plan cache
 
-<span class="version-tag">New in v2.2</span>: CockroachDB can use a cache for the query plans generated by the optimizer. This can lead to faster query execution since the database can reuse a query plan that was previously calculated, rather than computing a new plan each time a query is executed.
+<span class="version-tag">New in v2.2</span>: CockroachDB uses a cache for the query plans generated by the optimizer. This can lead to faster query execution since the database can reuse a query plan that was previously calculated, rather than computing a new plan each time a query is executed.
 
-The query plan cache is disabled by default. To enable it, execute the following statement:
+The query plan cache is enabled by default. To disable it, execute the following statement:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SET sql.query_cache.enabled=true;
+> SET CLUSTER SETTING sql.query_cache.enabled = false;
 ~~~
 
 {{site.data.alerts.callout_info}}
-The query plan cache is still under development and has the following limitations:  
-- The cache is only used for non-prepared statements (i.e., queries that correspond to the ["simple query" pgwire message](https://www.postgresql.org/docs/10/protocol-flow.html#id-1.10.5.7.4)).  
+The query plan cache is still under development and has the following limitation:  
 - The cache has simplistic memory management: it uses a fixed number of "slots" and rejects all plans above a certain size.  
```

Comment:
This is fixed too

Reply:
Oh nice!  Will remove that note altogether then.